react-blog/src/pages/Post/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import axios from 'axios';
+
+
+const Post = () => {
+    const [post, setPost] = useState({
+        id: 0,
+        title: "",
+        text: "",
+        image: "",
+    })
+
+    const params = useParams();
+    console.log(params);
+
+
+    useEffect(() => {
+      axios.get(`https://studapi.teachmeskills.by/blog/posts/${params.id}/`).then((response) => {
```

Comment:
Move it to our posts service file as an additional method

Reply:
@VVMig I have DMed you on Telegram about this